pkg/util/log/strip.go

```diff
@@ -72,6 +72,10 @@ func matchYAMLKey(key string) *regexp.Regexp {
 	return regexp.MustCompile(fmt.Sprintf(`(\s*%s\s*:).+`, key))
 }
 
+func matchYAMLKeyEnding(ending string) *regexp.Regexp {
+	return regexp.MustCompile(fmt.Sprintf(`(\s*(\w|_)*%s\s*:).+`, ending))
```

Comment:
nit: I got why you've called it this way but that's a little bit confusing while reading this, don't you think?